core/src/main/resources/hudson/Messages.properties

```diff
@@ -100,7 +100,8 @@ PluginManager.deprecationWarning=<strong>This plugin is deprecated.</strong> In
 PluginManager.insecureUrl=\
   You are using an insecure URL to download the plugin, use at your own risk!
 PluginManager.invalidUrl=\
-  You are using an invalid URL to download the plugin, only https and http (not recommended) are supported.
+  You are using an invalid URL to download the plugin, only https and http (not recommended) are supported. \
```

Comment:
we can not say "only http and https are supported" and then go on to say "oh but we support files as well".  The two statements are mutually exclusive :)